crates/common_utils/src/id_type/merchant_connector_account.rs

```diff
@@ -0,0 +1,23 @@
+use crate::errors::{CustomResult, ValidationError};
+
+crate::id_type!(
+    MerchantConnectorAccountId,
+    "A type for merchant_connector_id that can be used for merchant_connector_account ids"
+);
+crate::impl_id_type_methods!(MerchantConnectorAccountId, "merchant_connector_id");
+
+// This is to display the `MerchantConnectorAccountId` as MerchantConnectorAccountId(abcd)
+crate::impl_debug_id_type!(MerchantConnectorAccountId);
+crate::impl_default_id_type!(MerchantConnectorAccountId, "mca");
```

Comment:
Do we need the `Default` implementation, can we only use `GenerateId` implementation instead?